rigs/dummy6002.jl

```diff
@@ -53,7 +53,7 @@ LASER_OFF_TIME[rig_key] = 100 * Unitful.μs
 CAMERA_ON_TIME[rig_key] = 19526.0 * Unitful.ns #This is the worst jitter measured with Edge 5.5 and 4.2 cameras, see ImagineInterface issue #18
 CAMERA_OFF_TIME[rig_key] = 19526.0 * Unitful.ns #This is the worst jitter measured with Edge 5.5 and 4.2 cameras, see ImagineInterface issue #18
 
-PIEZO_RANGES[rig_key] = (0.0μm .. 400.0μm, 0.0V .. 10.0V)
+PIEZO_RANGES[rig_key] = (0..32767, 0.0μm .. 400.0μm, 0.0V .. 10.0V)
```

Comment:
For those that start at 0, should the top value be `Int(typemax(UInt16)) == 65535`? Or is the top bit still excluded?

Reply:
Excluded I think, Actually, it looks NIDAQ maps Int16 values to -10 to 10V (it means -32768..32767 to -10..10V).
The previous code doesn't use full range. it only use positive values. It maps 0..32767 to 0..10V.

Reply:
But pull request provide option to set the raw range freely other than fixed one 0..32767 for piezo.

Reply:
Thanks! Merge at will.